src/js/utils/is.js

```diff
@@ -9,7 +9,7 @@ const isObject = (input) => getConstructor(input) === Object;
 const isNumber = (input) => getConstructor(input) === Number && !Number.isNaN(input);
 const isString = (input) => getConstructor(input) === String;
 const isBoolean = (input) => getConstructor(input) === Boolean;
-const isFunction = (input) => getConstructor(input) === Function;
+const isFunction = (input) => typeof getConstructor(input) === 'function';
```

Comment:
I don't think your fix would have the intended effect: a constructor is always a function, so eg. `typeof (10).constructor === "function"` is true but `(10).constructor === Function` is false.

The same line of code causes async functions to not be executed https://github.com/sampotts/plyr/issues/2199
because `(async () => {}).constructor === AsyncFunction`.

I think this should work in all cases:
```suggestion
const isFunction = (input) => input instanceof Function;
```
```js
10 instancof Function //false
(() => {}) instancof Function // true
(async () => {}) instancof Function // true

const proxiedFunction = new Proxy(Function, { construct: (target, args) => new target(...args) });
new proxiedFunction() instanceof Function // true
```